From 2ab85db846c79b8f3ab710e7af44490c9734f057 Mon Sep 17 00:00:00 2001
From: Nirbheek Chauhan <nirbheek@centricular.com>
Date: Mon, 11 Jun 2018 07:02:08 +0530
Subject: [PATCH] meson: Always set G_MODULE_SUFFIX to so on macOS

This is what Autotools does, and it's what the entire world expects.

https://gitlab.gnome.org/GNOME/glib/issues/1413
---
 meson.build | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/meson.build b/meson.build
index 91b8721a1..07ff90174 100644
--- a/meson.build
+++ b/meson.build
@@ -1227,11 +1227,9 @@ glibconfig_conf.set('glongbits', long_size * 8)
 glibconfig_conf.set('gsizebits', sizet_size * 8)
 glibconfig_conf.set('gssizebits', ssizet_size * 8)
 
-# FIXME: maybe meson should tell us the libsuffix?
+# XXX: https://gitlab.gnome.org/GNOME/glib/issues/1413
 if host_system == 'windows'
   g_module_suffix = 'dll'
-elif host_system == 'darwin'
-  g_module_suffix = 'dylib'
 else
   g_module_suffix = 'so'
 endif
-- 
2.17.0

